ql/src/test/results/clientpositive/llap/subquery_multi.q.out

```diff
@@ -4769,7 +4769,7 @@ POSTHOOK: Input: default@tempty
 85768	almond antique chartreuse lavender yellow	Manufacturer#1	Brand#12	LARGE BRUSHED STEEL	34	SM BAG	1753.76	refull
 86428	almond aquamarine burnished black steel	Manufacturer#1	Brand#12	STANDARD ANODIZED STEEL	28	WRAP BAG	1414.42	arefully 
 90681	almond antique chartreuse khaki white	Manufacturer#3	Brand#31	MEDIUM BURNISHED TIN	17	SM CASE	1671.68	are slyly after the sl
-Warning: Shuffle Join MERGEJOIN[39][tables = [$hdt$_0, $hdt$_1, $hdt$_2]] in Stage 'Reducer 3' is a cross product
+Warning: Shuffle Join MERGEJOIN[39][tables = [$hdt$_0, $hdt$_2]] in Stage 'Reducer 3' is a cross product
```

Comment:
Need to figure out why `$hdt$_1` is omited...

Reply:
Same as below, the `$hdt$_1` does not participate in `Reducer 3`, as it produces no output after the semi join(`p_name IN (select p_name from part_null)`)